src/main/scala/no/ndla/articleapi/model/api/VisualElement.scala

```diff
@@ -15,8 +15,7 @@ import org.scalatra.swagger.runtime.annotations.ApiModelProperty
 import scala.annotation.meta.field
 
 @ApiModel(description = "Description of a foot note item")
-case class VisualElement(@(ApiModelProperty@field)(description = "The resource identifier") resource: String,
-                         @(ApiModelProperty@field)(description = "The type of visual element (can be an image, a video or a presentation)") `type`: String,
+case class VisualElement(@(ApiModelProperty@field)(description = "The resource identifier") content: String,
```

Comment:
Navnet er litt misvisende, siden det visuelle elementet representeres med en `embed` tag. Validatoren sjekker at den kun inneholder lovlig HTML.
Kan endre den til `HTML containing the visual element`, for eksempel?